api-changes.json

```diff
@@ -1,4 +1,23 @@
 {
+  "1.5.0": {
+    "revapi": {
+      "ignore": [
+        {
+          "code": "java.method.parameterTypeChanged",
```

Comment:
this is to be removed after this is cherry picked to 1.4.x, right?

Reply:
Yes.